scripts/get_info.py

```diff
@@ -12,91 +15,98 @@
-                hutch=ihutch.upper()
-                foundHutch=True
+            if hutch.find(ihutch.upper()) >= 0:
+                hutch = ihutch.upper()
+                foundHutch = True
                 break
         if not foundHutch:
             print('unknown_hutch')
             sys.exit()
 else:
-    hostname=socket.gethostname()
-    ip=socket.gethostbyname(hostname)
-    subnet=ip.split('.')[2]
-    for ihutch in hutches: #use the IP address to match the host to a hutch by subnet
+    hostname = socket.gethostname()
+    ip = socket.gethostbyname(hostname)
+    subnet = ip.split('.')[2]
+    for ihutch in hutches:  # use the IP address to match the host to a hutch by subnet
         if subnet in hutch_subnets.get(ihutch):
-            hutch=ihutch.upper()
-            foundHutch=True
+            hutch = ihutch.upper()
+            foundHutch = True
             break
     if not foundHutch:
         for ihutch in hutches:
-            if hostname.find(ihutch)>=0:
-                hutch=ihutch.upper()
-                foundHutch=True
+            if hostname.find(ihutch) >= 0:
+                hutch = ihutch.upper()
+                foundHutch = True
                 break
     if not foundHutch:
-        if hostname.find('psusr')>=0:
-            if hostname.find('psusr13')>=0:
-                hutch='XPP'
-            elif hostname.find('psusr21')>=0:
-                hutch='XCS'
-            elif hostname.find('psusr22')>=0:
-                hutch='CXI'
-            elif hostname.find('psusr23')>=0:
-                hutch='MEC'
-            elif hostname.find('psusr24')>=0:
-                hutch='MFX'
-            if hutch!='':
-                foundHutch=True
+        if hostname.find('psusr') >= 0:
+            if hostname.find('psusr13') >= 0:
+                hutch = 'XPP'
+            elif hostname.find('psusr21') >= 0:
+                hutch = 'XCS'
+            elif hostname.find('psusr22') >= 0:
+                hutch = 'CXI'
+            elif hostname.find('psusr23') >= 0:
+                hutch = 'MEC'
+            elif hostname.find('psusr24') >= 0:
+                hutch = 'MFX'
+            if hutch != '':
+                foundHutch = True
         else:
-            #then check current path
-            path=os.getcwd()
+            # then check current path
+            path = os.getcwd()
             for ihutch in hutches:
-                if path.find(ihutch)>=0:
-                    hutch=ihutch.upper()
-                    foundHutch=True
+                if path.find(ihutch) >= 0:
+                    hutch = ihutch.upper()
+                    foundHutch = True
                     break
-            if not foundHutch and path.find('xrt')+hostname.find('xrt')>=-1 or path.find('xtod')+hostname.find('xtod')>=-1:
-                hutch='LFE' #because we have so many names for the same subnet.
-                foundHutch=True
+            if (not foundHutch and path.find('xrt')+hostname.find('xrt') >= -1
```

Comment:
I'm slightly surprised this line passes flake8 with the lack of + spacing

Reply:
I'm also confused about this...